esmvalcore/local.py

```diff
@@ -421,14 +421,23 @@ def _select_drs(input_type: str, project: str, structure: str) -> list[str]:
     )
 
 
-@dataclass(order=True, frozen=True)
+@dataclass(order=True)
```

Comment:
```suggestion
@dataclass(order=True, frozen=True)
```
Can we keep this? Not sure it's terribly useful

Reply:
Nope, otherwise setting the `_regex_pattern` attribute won't work.